src/features/navbar/Navbar.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import {
+  AppBar,
+  Toolbar,
+  Typography,
+  Button,
+  IconButton,
+  Tooltip,
+} from '@material-ui/core';
+import MenuIcon from '@material-ui/icons/Menu';
+import { NavLink } from 'react-router-dom';
+import SideMenu from 'features/sidemenu/SideMenu';
+
+import MainLogo from 'assets/img/MainPageLogo.jpg';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    flexGrow: 1,
+  },
+  menuButton: {
+    marginRight: theme.spacing(2),
+  },
+  title: {
+    flexGrow: 1,
+  },
+}));
+
+export default function Navbar(): JSX.Element {
+  const classes = useStyles();
+  const [drawerState, setDrawerState] = useState(false);
+
+  const handleToggleSideMenu = () => {
+    setDrawerState(!drawerState);
```

Comment:
```suggestion
    setDrawerState(!drawerState);
```
Здесь это наверное не сильно влияет, но вроде как правильнее менять состояние, основанное на предыдущем состоянии через функцию:
setDrawerState((s) => !s);

Reply:
ОК